nixos/modules/services/networking/syncthing.nix

```diff
@@ -117,6 +115,14 @@ in
 
   config = mkIf cfg.enable {
 
+    networking.firewall = {
```

Comment:
I would prefer not to do that, because I run Syncthing on non-default ports and port configuration is possible only via the web interface.